apps/synapse-portal-framework/src/pages/FileEntityPage/SynapseFileEntityPageProperties.tsx

```diff
@@ -134,7 +140,6 @@ const SynapseFileEntityPageProperties = ({
     { key: 'versionLabel', label: 'Version' },
     { key: 'contentMd5', label: 'MD5' },
     { key: 'storageLocationId', label: 'Storage Location' },
-    { key: 'parentId', label: 'Contained in Project' },
```

Comment:
Note: parentId is the parent container.  It might be the Project (if the file was uploaded to the Project), but in many cases it will be a Folder (which may be a nested subfolder in the Project).